generated/java/gapic-google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/v2/BigtableClient.java

```diff
@@ -0,0 +1,636 @@
+ *   String formattedTableName = BigtableClient.formatTableName("[PROJECT]", "[INSTANCE]", "[TABLE]");
+ *   ByteString rowKey = ByteString.copyFromUtf8("");
+ *   List&lt;Mutation&gt; mutations = new ArrayList&lt;&gt;();
+ *   MutateRowResponse response = bigtableClient.mutateRow(formattedTableName, rowKey, mutations);
+ * }
+ * </code>
+ * </pre>
+ *
+ * <p>Note: close() needs to be called on the bigtableClient object to clean up resources such as
+ * threads. In the example above, try-with-resources is used, which automatically calls close().
+ *
+ * <p>The surface of this class includes several types of Java methods for each of the API's
+ * methods:
+ *
+ * <ol>
+ *   <li> A "flattened" method. With this type of method, the fields of the request type have been
+ *       converted into function parameters. It may be the case that not all fields are available as
+ *       parameters, and not every API method will have a flattened method entry point.
+ *   <li> A "request object" method. This type of method only takes one parameter, a request object,
+ *       which must be constructed before the call. Not every API method will have a request object
+ *       method.
+ *   <li> A "callable" method. This type of method takes no parameters and returns an immutable API
+ *       callable object, which can be used to initiate calls to the service.
+ * </ol>
+ *
+ * <p>See the individual methods for example code.
+ *
+ * <p>Many parameters require resource names to be formatted in a particular way. To assist with
+ * these names, this class includes a format method for each type of name, and additionally a parse
+ * method to extract the individual identifiers contained within names that are returned.
+ *
+ * <p>This class can be customized by passing in a custom instance of BigtableSettings to create().
+ * For example:
+ *
+ * <p>To customize credentials:
+ *
+ * <pre>
+ * <code>
+ * BigtableSettings bigtableSettings =
+ *     BigtableSettings.newBuilder()
+ *         .setCredentialsProvider(FixedCredentialsProvider.create(myCredentials))
+ *         .build();
+ * BigtableClient bigtableClient =
+ *     BigtableClient.create(bigtableSettings);
+ * </code>
+ * </pre>
+ *
+ * To customize the endpoint:
+ *
+ * <pre>
+ * <code>
+ * BigtableSettings bigtableSettings =
+ *     BigtableSettings.newBuilder()
+ *         .setTransportProvider(BigtableSettings.defaultGrpcTransportProviderBuilder()
+ *             .setChannelProvider(BigtableSettings.defaultGrpcChannelProviderBuilder()
+ *                 .setEndpoint(myEndpoint)
+ *                 .build())
+ *             .build())
+ *         .build();
+ * BigtableClient bigtableClient =
+ *     BigtableClient.create(bigtableSettings);
+ * </code>
+ * </pre>
+ */
+@Generated("by GAPIC v0.0.5")
+@BetaApi
+public class BigtableClient implements BackgroundResource {
+  private final BigtableSettings settings;
+  private final BigtableStub stub;
+
+  private static final PathTemplate TABLE_PATH_TEMPLATE =
```

Comment:
It seems to me you need to configure resource name class for these.

Reply:
Blocked on cl/173174433.

Reply:
Fixed.